### PR TITLE
Ft user states

### PR DESCRIPTION
**Pull request by FelixWambiri**
**Description of Task to be completed?**
* Fix errors occurring when deploying to Heroku and incorporate the changes from develop branch. 

**How should this be manually tested?**
* Login as an admin with password admin
* Create a new gym and give it a name of your choice
* Add a few members to the gym.
* As an administrator use your permissions to deactivate a few users. 
* Go back to gym overview and click on the all, active and deactivated buttons to view all users.

**What are the relevant pivotal tracker stories?**
* #157507861

**Background context**
* At the moment this feature is failing with an error on heroku as some changes have not been icorporated.

### CODE REVIEW

**Comment by mutoromike** [comment]
Request for reviews.

**Review by mutoromike — APPROVED** [review]
Good work fixing the errors!

**Review by meshnesh — APPROVED** [review]

**Review by frankip — APPROVED** [review]
good job 
👍

**Review by Kalela — APPROVED** [review]
Good job

File: settings.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from wger.settings_global import *
```
Comment (frankip) [inline settings.py:4]: did you create a new settings file

**Review by betsybeth — APPROVED** [review]
Good job

**Pull request merged**